python/whylogs/api/store/sqlite_store.py

```diff
@@ -0,0 +1,65 @@
+import logging
+import os
+import sqlite3
+from datetime import datetime
+from typing import Optional, Union
+
+from whylogs.api.store import ProfileStore
+from whylogs.api.store.query import DateQuery, ProfileNameQuery
+from whylogs.core import DatasetProfile, DatasetProfileView
+
+logger = logging.getLogger(__name__)
+
+
+class SQLiteStore(ProfileStore):
+    def __init__(self, connection: Optional[sqlite3.Connection] = None):
+        self._db_location = os.getenv("SQLITE_STORE_LOCATION") or ":memory:"
```

Comment:
Would be great to start a convention early for a Config type that centralizes all of the different env configs

Reply:
done on service-side for now, not to blow up whylogs' (i figure it needs to be more discussed, since other things could also benefit from a central config)